pyaerocom/aeroval/fairmode_stats.py

```diff
@@ -13,10 +13,10 @@
 import numpy as np
 
 SPECIES = dict(
-    concno2=dict(UrRV=0.24, RV=200, alpha=0.2),
-    conco3mda8=dict(UrRV=0.18, RV=120, alpha=0.79),
-    concpm10=dict(UrRV=0.28, RV=50, alpha=0.25),
-    concpm25=dict(UrRV=0.36, RV=25, alpha=0.5),
+    concno2=dict(UrRV=0.24, RV=200, alpha=0.2, freq="hourly"),
```

Comment:
For clarity, can we make all `freq`uencies a `TsType`? See https://github.com/metno/pyaerocom/pull/1267/files